src/marlin_stubs/pause/pause.cpp

```diff
@@ -67,13 +67,13 @@
     (!ENABLED(PREVENT_COLD_EXTRUSION)) || \
     ENABLED(ADVANCED_PAUSE_CONTINUOUS_PURGE) || \
     BOTH(FILAMENT_UNLOAD_ALL_EXTRUDERS, MIXING_EXTRUDER) || \
-    ENABLED(HOST_ACTION_COMMANDS) || \
-    ENABLED(HOST_PROMPT_SUPPORT) || \
     ENABLED(SDSUPPORT)
 #error unsupported
 #endif
 // clang-format on
 
+PauseMenuResponse pause_menu_response;
```

Comment:
what is this and how is it related to the octoprint screen?

Reply:
After added Marlin's Host features in need to be included, because it's needed in host_actions as a response handler. I can't make a build without it.

Reply:
ok